android/java/res/values/brave_colors.xml

```diff
@@ -413,4 +413,10 @@
 
     <color name="location_bar_background_color_for_ntp">#f2f2f2</color>
     <color name="toolbar_background_color_for_ntp">@android:color/white</color>
+
+    <color name="primitive_blurple_20">#251491</color>
```

Comment:
@aguscruiz i have used these colours from figma ? can you check if it's updated ?

Reply:
Figma is using primitive/blurple/10, not 20 
That seems to be the issue with the background

https://www.figma.com/design/DSDkzFJ4NdBKLKkf3tDdnE/%F0%9F%94%90-VPN?node-id=3054-395&t=CaQUX6jQ9xft2jiQ-1

<img width="739" alt="image" src="https://github.com/user-attachments/assets/16e25029-898d-4cce-beed-4c93c0408260">


![image](https://github.com/user-attachments/assets/d44c5a54-5d85-4bce-a28f-6928dcf6d81f)